test/e2e/service_serving_ca_bundle_exists_test.go

```diff
@@ -0,0 +1,71 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/remotecommand"
+
+	"github.com/kube-reporting/metering-operator/test/reportingframework"
+)
+
+func testReportOperatorServiceCABundleExists(t *testing.T, rf *reportingframework.ReportingFramework) {
```

Comment:
Going forward, we'll most like want to iterate over the operand containers, exec into each container, and ensure the service-ca.crt exists.